apps/files/src/components/Collaborators/SharedFilesList.vue

```diff
@@ -136,6 +137,31 @@ export default {
   methods: {
     ...mapActions('Files', ['loadFolderSharedFromMe', 'loadFolderSharedWithMe', 'setFilterTerm', 'pendingShare']),
 
+    prepareCollaborators (shares) {
+      let hasLink = false
+      const results = []
+      shares.forEach(share => {
+        if (share.shareType === shareTypes.link) {
+          if (!hasLink) {
```

Comment:
Maybe add a comment here to make it clear why it's needed?

Reply:
added in method doc